### PR TITLE
chore(release): add release notes for armory operator.

### PR DESCRIPTION
**Pull request by cristhian-castaneda**

### CODE REVIEW

**Review by dogonthehorizon — APPROVED** [review]
LGTM, though I'd wait for the docs team to approve before merging!

**Review by 317brian — APPROVED** [review]
Thanks @cristhian-castaneda! I removed the redirects cause they're not necessary since the page is new. The existing redirects are there because the URL format changed between the old docs site and new docs site. 

Really appreciate you being thorough though!

**Pull request merged**
